rq/worker.py

```diff
@@ -564,6 +564,7 @@ def perform_job(self, job):
             started_job_registry = StartedJobRegistry(job.origin, self.connection)
 
             try:
+                job.started_at = utcnow()
```

Comment:
Could you please move this assignment to `prepare_job_execution`? This will persist the information into Redis along with `job.status`.